opentracing-impl-java8/src/main/java/io/opentracing/AbstractSpan.java

```diff
@@ -127,18 +123,46 @@ public String getBaggageItem(String key) {
     }
 
     @Override
-    public final Span log(String message, /* @Nullable */ Object payload) {
+    public final Span log(String event) {
+        return log(System.nanoTime() / 1000, event);
```

Comment:
this should be `System.currentTimeMillis() * 1000` instead. 

`nanoTime()`:

> This method can only be used to measure elapsed time and is not related to any other notion of system or wall-clock time. The value returned represents nanoseconds since some fixed but arbitrary origin time (perhaps in the future, so values may be negative). The same origin is used by all invocations of this method in an instance of a Java virtual machine; other virtual machine instances are likely to use a different origin.